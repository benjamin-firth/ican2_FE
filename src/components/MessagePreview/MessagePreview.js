import React from 'react';
import './MessagePreview.scss';

const MessagePreview = ({ otherMessenger }) => {

  return (
    <form>
      <p>This a message between you and {otherMessenger}.</p>
    </form>
  );
}

export default MessagePreview;
