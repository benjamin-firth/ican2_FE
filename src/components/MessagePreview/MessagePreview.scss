@import '../../variables.scss';

.message-preview {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 2vw;
  border: 5px solid $secondarycolor;
  padding: 1vw;
  width: 18vw;

  img {
    border-radius: 50%;
    width: 15vw;
    height: 15vw;
  }

  button {
    border: 5px solid $secondarycolor;
  }
}
