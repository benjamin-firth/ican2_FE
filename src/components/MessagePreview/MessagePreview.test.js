import React from 'react';
import { shallow } from 'enzyme';
import MessagePreview from './MessagePreview';

jest.mock("react-redux", () => ({
  useSelector: () => 'mockState',
  useDispatch: () => jest.fn()
}));

describe('MessagePreview', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <MessagePreview />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
