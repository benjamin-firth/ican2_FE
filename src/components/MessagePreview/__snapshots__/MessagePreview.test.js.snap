// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagePreview should match the snapshot 1`] = `
<section
  className="message-preview"
/>
`;
